Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:18-bullseye
```

Comment:
and yes, this works
```suggestion
FROM node:18
```

Reply:
I know it does, but the default one is Alpine Linux and it's harder to maintain it because it's less popular